Dockerfile.example

```diff
@@ -0,0 +1,70 @@
+# ubuntu trusty base image for building other images within N & S
+#
+# build
+# $ docker build -f Dockerfile.ubuntu-trusty-base -t nens/ubuntu-trusty-base:latest .
+#
+# release
+# $ docker tag <image_id> dockrepo.lizard.net/nens/ubuntu-trusty-base:latest
+#
+# push it to the registry
+# $ docker push dockrepo.lizard.net/nens/ubuntu-trusty-base:latest
+#
+
+FROM ubuntu:xenial
+
+MAINTAINER 3Di <3Di@nelen-schuurmans.nl>
+
+# change the date to force rebuilding the whole image
+ENV REFRESHED_AT 20160531
+
+# system dependencies
+# Sympy installs a lot of fonts, latex stuff, etc., which we don't want, so
+# that is run with the --no-install-recommends option.
+RUN apt-get update && apt-get install -y \
+    python-software-properties \
+    wget \
+    build-essential \
+    git \
+    libevent-dev \
+    libfreetype6-dev \
+    libgeos-dev \
+    libhdf5-serial-dev \
+    libpng12-dev \
+    mercurial \
+    python-dev \
+    python-lxml \
+    python-mapnik \
+    python-numpy \
+    python-pip \
+    python-pyproj \
+    python-pysqlite2 \
+    python-psycopg2 \
+    python-scipy \
+    spatialite-bin \
+    gdal-bin \
+    libgdal1i \
+    libnetcdf-dev \
+    netcdf-bin \
+    python-gdal \
+    vim \
+&& apt-get install -y --no-install-recommends \
+    python-sympy \
+&& apt-get clean -y && rm -rf /var/lib/apt/lists/*
+
+# pip packages
+RUN pip install \
+    matplotlib \
+    six
+
+# Add 'buildout' group and user with the same gid and uid as the host user.
+# TODO: for now gid:uid is hardcoded to 1000:1000, change this if it isn't correct for you.
+RUN groupadd --gid 1000 buildout && \
```

Comment:
Het is geinspireerd op het docker django voorbeeld: https://docs.docker.com/compose/django/

In feite kun je met `docker-compose run` je custom commando's draaien nadat je image is gebouwd, dus daarmee bijv `buildout`, `syncdb`, etc. draaien